src/handlebars/testimonials.handlebars

```diff
@@ -97,9 +93,12 @@
       </div>
 
       <div class="clients">
-        <img src="https://www.tirasa.net/images/logo.svg" alt="Tirasa logo" />
+        <img src="https://upload.wikimedia.org/wikipedia/commons/9/9d/Soci%C3%A9t%C3%A9_G%C3%A9n%C3%A9rale.svg" alt="Societe Generale logo" />
```

Comment:
What happened to Tirasa?

Reply:
I've switched them out

Reply:
They still appear on the full users page